docs/source/openmw-cs/tour.rst

```diff
@@ -58,7 +58,7 @@ detached panel can be re-attached to a window by dragging it by the title bar
 on top of the window.
 
 Now let's look at the panel itself: we have a filter text field, a very large
-table and a status bar. The filter will be very useful when we want to find an
+table, and a status bar. The filter will be very useful when we want to find an
```

Comment:
You're quite right, I must have been reading the following line incorrectly.

The unnecessary Oxford comma still stands though. :)